modules/settings/src/main/native/android/c/Settings.c

```diff
@@ -92,10 +79,10 @@ JNIEXPORT void JNICALL Java_com_gluonhq_attach_settings_impl_AndroidSettingsServ
     ATTACH_DALVIK();
     jstring dkey = (*dalvikEnv)->NewStringUTF(dalvikEnv, keyChars);
     jstring dvalue = (*dalvikEnv)->NewStringUTF(dalvikEnv, valueChars);
-    if (debugSettings) {
+    if (debugUtil) {
         ATTACH_LOG_FINE("Storing settings for = %s, %s\n", keyChars, valueChars);
     }
-    (*dalvikEnv)->CallObjectMethod(dalvikEnv, jDalvikSettingsService, jSettingsServiceStore, dkey, dvalue);
+    (*dalvikEnv)->CallVoidMethod(dalvikEnv, jDalvikSettingsService, jSettingsServiceStore, dkey, dvalue);
```

Comment:
This seems to be unrelated to debug, but it was a bug apparently?

Reply:
Indeed, that is is the case. It was crashing on Android, so I couldn't test the changes for this PR. It should have been part of a different PR.

Reply:
Ok, fixed. Change is moved to: https://github.com/gluonhq/attach/pull/132